supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift.adoc

```diff
@@ -429,6 +429,18 @@ Ensure that you provide context about which type of quick start you are referrin
 
 *See also*:
 
+[discrete]
+[[red-hat-openshift-cluster-manager]]
+==== image:images/yes.png[yes] Red Hat OpenShift Cluster Manager (noun)
+*Description*: A managed service for Red Hat OpenShift that lets users create, subscribe, and manage different types of OpenShift clusters from a single user interface. After first mention, you can use OpenShift Cluster Manager. link:https://console.redhat.com/openshift[OpenShift Cluster Manager] is part of Red Hat Hybrid Cloud Console.
+
+*Use it*: yes
+
+*Incorrect forms*: OCM, Cluster Manager, the OpenShift Cluster Manager, the OpenShift Cluster Manager site
+
+*See also*: The _Technologies_ sheet in the link:https://docs.google.com/spreadsheets/d/1DLS_lS3VKidgZIvcLmLp9BoiqptkvqHWfe1D5FD2kfk/edit#gid=1375785039[Official Red Hat product and solution names list]
```

Comment:
@dayleparker Can you remove this sentence/link here? The SSG is public, and this is an internal link.

Reply:
@bergerhoffer You bet! Sorry to miss considering that - this is my first PR for the style guide. Should I also remove the related Jira link? As a writer, I like having this sort of additional information, but I wasn't sure if it was something we do in this guide.

Reply:
Don't mind me, I only included that Jira in the PR. :)